src/helm/benchmark/presentation/run_specs_openchatkit.conf

```diff
@@ -0,0 +1,87 @@
+# Main `RunSpec`s for the benchmarking.
```

Comment:
You shouldn't need this file if you just want to run the standard core scenarios on a given model.